Library/Homebrew/download_strategy.rb

```diff
@@ -572,6 +568,10 @@ def curl_output(*args, **options)
     super(*_curl_args, *args, **_curl_opts, **options)
   end
 
+  def curl_head(*args, **options)
+    super(*_curl_args, *args, **_curl_opts, **options)
```

Comment:
I'm not sure I understand why this is exactly the same as `curl_output` above?

Reply:
Yeah, this may not be needed since `curl_head` calls `curl_output`.